runtime/bench_standard_transactions_test.go

```diff
@@ -0,0 +1,977 @@
+			},
+			Context{
+				Interface:   runtimeInterface,
+				Location:    nextTransactionLocation(),
+				Environment: environment,
+				UseVM:       useVM,
+			},
+		)
+		require.NoError(b, err)
+
+		// Deploy Crypto contract
+		err = runtime.ExecuteTransaction(
+			Script{
+				Source: DeploymentTransaction("Crypto", []byte(realCryptoContract)),
+			},
+			Context{
+				Interface:   runtimeInterface,
+				Location:    nextTransactionLocation(),
+				Environment: environment,
+				UseVM:       useVM,
+			},
+		)
+		require.NoError(b, err)
+
+		// Setup both user accounts for Flow Token
+		for _, address := range []common.Address{
+			senderAddress,
+			receiverAddress,
+		} {
+			signerAccount = address
+
+			err = runtime.ExecuteTransaction(
+				Script{
+					Source: []byte(realSetupFlowTokenAccountTransaction),
+				},
+				Context{
+					Interface:   runtimeInterface,
+					Location:    nextTransactionLocation(),
+					Environment: environment,
+					UseVM:       useVM,
+				},
+			)
+			require.NoError(b, err)
+		}
+
+		// Mint 1000 FLOW to sender
+		mintAmount, err := cadence.NewUFix64("100000000000.0")
+		require.NoError(b, err)
+
+		signerAccount = contractsAddress
+
+		err = runtime.ExecuteTransaction(
+			Script{
+				Source: []byte(realMintFlowTokenTransaction),
+				Arguments: encodeArgs([]cadence.Value{
+					cadence.Address(senderAddress),
+					mintAmount,
+				}),
+			},
+			Context{
+				Interface:   runtimeInterface,
+				Location:    nextTransactionLocation(),
+				Environment: environment,
+				UseVM:       useVM,
+			},
+		)
+		require.NoError(b, err)
+
+		// Set signer account to sender for benchmark transactions
+		signerAccount = senderAddress
+
+		return runtime, nextTransactionLocation, runtimeInterface
+	}
+
+	for _, transaction := range testTransactions {
+
+		b.Run(transaction.Name, func(b *testing.B) {
+			// Create fresh runtime and storage for this sub-benchmark
+			runtime, nextTransactionLocation, runtimeInterface := setupRuntime()
+
+			for b.Loop() {
+				b.StopTimer()
+				// set up everything for the transaction
+				var err error
+				if transaction.Setup != "" {
+					err = runtime.ExecuteTransaction(
+						Script{
+							Source:    []byte(transaction.Setup),
+							Arguments: nil,
+						},
+						Context{
+							Interface:   runtimeInterface,
+							Location:    nextTransactionLocation(),
+							Environment: environment,
+							UseVM:       useVM,
+						},
+					)
+					require.NoError(b, err)
+				}
+				source := []byte(transaction.Body)
+				location := nextTransactionLocation()
+				b.StartTimer()
+
+				err = runtime.ExecuteTransaction(
+					Script{
+						Source:    source,
+						Arguments: nil,
+					},
+					Context{
+						Interface:   runtimeInterface,
+						Location:    location,
+						Environment: environment,
+						UseVM:       useVM,
+					},
+				)
+
+				b.StopTimer()
+				require.NoError(b, err)
+				b.StartTimer()
```

Comment:
Why does it need to start the timer `b.StartTimer()` here and stop it at the beginning of the loop? Can't we simply remove both?